### PR TITLE
Edit caption to be more descriptive

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @1tejaswini1
- [ ] @GVEERABABU
- [ ] @Radhikamuvva
- [ ] @aharshit93
- [ ] @akkp-cts
- [ ] @bsuppala
- [ ] @ericalloyd
- [ ] @mvikas99
- [ ] @prasobhHERE
- [ ] @rasrinivasa
- [ ] @rkumarreddy
- [ ] @shahim22
- [ ] @stevenvelasquez
- [ ] @vbasvabovina
- [ ] @vikas-khindkar
- [ ] @auharrison-clgx

### PR DESCRIPTION
**Pull request by auharrison-clgx**
Resolves #2

### CODE REVIEW

File: _slides/17-auharrison-clgx.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hi there!
```
Comment (stevenvelasquez) [inline _slides/17-auharrison-clgx.md:6]: Looks good

**Review by stevenvelasquez — APPROVED** [review]
Looks good

File: _slides/17-auharrison-clgx.md (hunk 1 above)
Comment (vbasvabovina-clgx) [inline _slides/17-auharrison-clgx.md:6]: Thank you

**Review by vbasvabovina-clgx — APPROVED** [review]

**Review by shahim22 — COMMENTED** [review]
Nice one

**Pull request merged**